.github/workflows/docker-image.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     - name: List of all Github Actions env vars
       run: env
     - name: Build the Docker image
-      run: make -C docker ${{ matrix.docker-image }} REPO=${{ github.repository }} BRANCH=${{ github.head_ref }}
+      run: make -C docker ${{ matrix.docker-image }} REPO="${{ github.repository }}" BRANCH="${{ github.head_ref }}"
```

Comment:
Does an image name never need the quotes?

Reply:
We control that variable.